examples/jsm/loaders/3DMLoader.js

```diff
@@ -1241,7 +1242,7 @@ Rhino3dmLoader.Rhino3dmWorker = function () {
 
 		}
 
-		if ( geometry ) {
+		if ( ( geometry && ! Array.isArray( geometry ) ) || ( Array.isArray( geometry ) && geometry.length ) ) {
```

Comment:
How about this:
```js
if ( Array.isArray( geometry ) === false || geometry.length > 0 ) ) {
```

Reply:
Thanks! Updated. Tested on several models and works as expected.